test/e2e/e2e_suite.go

```diff
@@ -211,6 +213,33 @@ var _ = Describe("operator-sdk", func() {
 				"--timeout", "4m")
 			_, err = tc.Run(cleanupPkgManCmd)
 			Expect(err).NotTo(HaveOccurred())
+
+			By("running basic scorecard tests")
```

Comment:
You will have to run these steps before `cleanup packagemanifests` is called.

Reply:
It shows done closing it as solved.